### PR TITLE
NETOBSERV-656 follow-up: add chart tooltips

### PR DESCRIPTION
**Pull request by jotak**
[NETOBSERV-656](https://issues.redhat.com//browse/NETOBSERV-656)

### CODE REVIEW

File: web/locales/en/plugin__netobserv-plugin.json
Comment (jotak) [inline web/locales/en/plugin__netobserv-plugin.json:135]: @memodi what do you think about those texts? Are they good enough, or would you reformulate?

Comment (jotak) [reply web/locales/en/plugin__netobserv-plugin.json]: (I've changed them, cf last commit)

**Comment by jotak** [comment]
I've also added tooltips on donut charts titles:
Capture d’écran du 2022-11-09 10-39-53

Capture d’écran du 2022-11-09 10-39-39

**Review by jpinsonneau — APPROVED** [review]
LGTM in terms of code

Let's wait for more feedback on text contents before merge

**Comment by memodi** [comment]
Thanks for adding tool tips, looks good!

/qe-approved

**Pull request merged**
